README.md

```diff
@@ -4,6 +4,16 @@ A repository for developing the User Interface for receiving data and controllin
 
 ## Packages Info
 
+### Whole Launch
+
+1.	Open your terminal and navigate to your working directory (for example, ros2_ws).
+2.	Clone the latest version of the MRCS: `git@github.com:GonzagaRobotics/ManualRoverControlSystem.git`
+3.	Once the repository is cloned, navigate to the top of the ros2_ws directory and build the project using Colcon by running the command: `colcon build`. If an error occurs, try again until it works.
+4.  Source from ros_ws: `. install/setup.bash`
+4.	When the build is complete and you're ready to launch, run the command: `ros2 launch src/ManualRoverControlSystem/WholeLaunch/launch/auto_launch.py`
```

Comment:
Consider breaking up "Whole Launch" into "Build" and "Launch", to avoid having to use qualifiers in your instructions. You could do:
## Build
1. do this
2. do that
3. run `./install.sh` to build

## Launch
1. do this
2. do that
3. run `./launch.sh` to launch

Again, Start each line of instruction with a verb and keep each step to only a few words. If you have more detail that the user doesn't ABSOLUTELY need, put it in a note as a bullet point under the step, not in the step itself.